src/app/public/styles/themes/modern/_compat/_mixins.scss

```diff
@@ -15,24 +23,69 @@
     color: $sky-text-color-default;
   }
 
+  &:focus {
+    border-bottom: solid 3px $sky-theme-modern-border-color-neutral-medium;
+  }
+
   .sky-btn-tab-close,
   .sky-tab-header-count {
     color: $sky-text-color-deemphasized;
   }
 }
 
+@mixin sky-theme-modern-btn-tab-dropdown-item {
+  &:hover:not(.sky-btn-tab-disabled):not(.sky-btn-tab-selected),
+  &:focus {
+    background-color: #fff;
+    border-bottom: none;
+    border-left: solid 3px $sky-theme-modern-border-color-neutral-medium;
+    outline: none;
+    padding-bottom: $sky-tab-btn-padding-top-bottom;
+    padding-left: $sky-tab-btn-padding-left-right - 3px;
+  }
+}
+
 @mixin sky-theme-modern-btn-tab-selected-dropdown-item {
+  color: $sky-text-color-default;
+  background-color: $sky-theme-modern-background-color-neutral-medium;
   border-bottom: none;
   border-left: solid 3px $sky-highlight-color-info;
   padding-bottom: $sky-tab-btn-padding-top-bottom;
   padding-left: $sky-tab-btn-padding-left-right - 3px;
+
+  &:focus {
+    border-left: solid 3px $sky-theme-modern-border-color-neutral-medium;
+    color: $sky-text-color-default;
+  }
 }
 
 @mixin sky-theme-modern-dark-btn-tab-selected {
   color: #fff;
 
+  &:hover {
+    color: #fff;
+  }
+
   .sky-btn-tab-close,
   .sky-tab-header-count {
     color: #efefef;
   }
 }
+@mixin sky-theme-modern-dark-btn-tab-dropdown-item {
+  background-color: #252a2e;
```

Comment:
Should this background color be stored in a variable for reuse?

Reply:
Eventually, but dark mode is in super-experimental mode right now and may be removed, so I'm trying not to create too many artifacts that relate to it yet.

Reply:
Sounds good.